ecl2df/common.py

```diff
@@ -621,3 +622,46 @@ def stack_on_colnames(dframe, sep="@", stackcolname="DATE", inplace=True):
     del dframe["level_0"]
     dframe.index.name = ""
     return dframe
+
+
+def parse_zonemapfile(filename: str):
+    """Return a dictionary from (int) K layers in the simgrid to strings
+
+    Typical usage is to map from grid layer to zone names.
+
+    The layer filename must currently follow format::
```

Comment:
```suggestion
    The layer filename must currently follow format:
```

Reply:
The double colon is a formatting syntax, making the following paragraph formatted verbatim.